DIGNDB.App.SmitteStop.API/Attributes/UploadKeysAuthorizationAttribute.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System;
+using DIGNDB.App.SmitteStop.API.Contracts;
+using AnonymousTokens.Server.Protocol;
+using AnonymousTokens.Core.Services.InMemory;
+
+namespace DIGNDB.App.SmitteStop.API.Attributes
+{
+    public class UploadKeysAuthorizationAttribute : ActionFilterAttribute
+    {
+        private readonly IJwtValidationService _jwtValidationService;
+        private readonly IAnonymousTokenKeySource _anonymousTokenKeySource;
+        private readonly ITokenVerifier _tokenVerifier = new TokenVerifier(new InMemorySeedStore());
```

Comment:
The InMemorySeedStore must be replaced by a real implementation.

Looking at the existing JwtTokenReplayAttackService, it looks like we can make an implementation of ISeedStore using the JwtToken-entity and IJwtTokenRepository, as these are just an ID and an expiry. Then this implementation just needs access to a config for the duration of each key pair, and set the expiry accordingly when creating new records.

Preferably, we should either rename the JwtToken-entity, or add a new entity for the anonymous token, but it should work just keeping it as is as well. Thought @jhannes @matishadow @OsirisTerje ?